File: web-deteccion/src/app/royal/modulos/framework-comun/sy/dominio/filtro/FiltroComunSyTipoDocumento.ts
import { DominioPaginacion } from '../../../../../framework/modelo/generico/DominioPaginacion';


export class FiltroComunSyTipoDocumento {
    constructor() {
        this.paginacion = new DominioPaginacion();
    }
    paginacion: DominioPaginacion;

    codigo: string;    
    estado: string;
    comentarios: string;
    nombre: string;
}

File: blinkeye-framewk-comun-dominio/src/main/java/net/royal/spring/sistema/dominio/filtro/FiltroComunSyTipoDocumento.java
package net.royal.spring.sistema.dominio.filtro;

import net.royal.spring.framework.modelo.generico.DominioPaginacion;

public class FiltroComunSyTipoDocumento {
	private String codigo;
	private String comentarios;
	private String nombre;
	private String estado;
	
	private DominioPaginacion paginacion;

	public String getCodigo() {
		return codigo;
	}

	public void setCodigo(String codigo) {
		this.codigo = codigo;
	}

	public String getComentarios() {
		return comentarios;
	}

	public void setComentarios(String comentarios) {
		this.comentarios = comentarios;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getEstado() {
		return estado;
	}

	public void setEstado(String estado) {
		this.estado = estado;
	}

	public DominioPaginacion getPaginacion() {
		return paginacion;
	}

	public void setPaginacion(DominioPaginacion paginacion) {
		this.paginacion = paginacion;
	}
	
	
}
